src/types/mapReduce.ts

```diff
@@ -0,0 +1,23 @@
+import { Document, MapReduceOptions, ObjectId } from 'mongodb'
+import { TsFilter } from './filter'
+import { TsSort } from './sort'
+
+// MapReduceOptions output type contains inline, which probably should be string?
+export type MapReduceOutput =
```

Comment:
Couldn't comprehend why mongodb have type of `out` as `inline' | { inline: 1 } | { replace: string } | { merge: string } | { reduce: string }`. 

Like this is valid

```
mapReduce(
  function(){ emit(this.Name,1)},
  function(key, values) {return Array.sum(values)},
  {query:{Marks:{$gt:70}},out: 'Name_Total'}
)
```

From my understanding, out should've been 

```
out?: string | { inline: 1 } | { replace: string } | { merge: string } | { reduce: string }
```